src/capi/scdef.rs

```diff
@@ -74,6 +74,16 @@ pub enum GFX_LAYER
 
 	/// Skia backend with OpenGL rendering.
 	SKIA_OPENGL = 5,
+
+  SKIA_VULKAN = 6,
+
+  #[cfg(target_os = "macos")]
+  SKIA_METAL = 7,
+
+	/// A hardware Direct2D mode (Windows only).
+	#[cfg(windows)]
+	SKIA_DX12 = 8,
```

Comment:
Are they available in SDK?

I don't see them neither in 
https://github.com/c-smile/sciter-js-sdk/blob/main/include/sciter-x-types.h#L20-L28
nor in 
https://github.com/c-smile/sciter-sdk/blob/master/include/sciter-x-types.h#L32-L40

Reply:
sorry, i find them from here: https://sciter.com/vulkan-metal-dx12/

Reply:
Yeah, I remember that post. But I have no idea whether it is actually implemented or not.